.github/workflows/run-tests.yml

```diff
@@ -75,7 +74,16 @@ jobs:
           path: ./ophys_testing_data
           key: ophys-datasets-042023-${{ matrix.os }}-${{ steps.ophys.outputs.HASH_OPHYS_DATASET }}
 
-      - name: Run full pytest with coverage
+      - name: Run full pytest (macos-latest)
+        if: ${{ matrix.os == 'macos-latest' }}
+        run: |
+          pytest -n auto --dist loadscope \
+            --ignore=tests/test_scan_image_tiff.py \
+            --ignore=tests/test_scanimage_utils.py \
+            --ignore=tests/test_scanimagetiffimagingextractor.py \
```

Comment:
Were you not able to get a skip to work well in the test suite itself? (like `pytest.mark.skipif`)?

Reply:
The cli-level ignore solution is more dry.  I am not particularly interested in adding pytest.mark.skipif to every test in tests/test_scanimagetiffimagingextractor.py (and then removing them if/when scanimage adds m1 mac support).

Reply:
It's just as many lines to type and remove, and is easier to catch from source code in tests than to notice the split in CI